keps/prod-readiness/sig-node/4438.yaml

```diff
@@ -0,0 +1,5 @@
+kep-number: 4438
+alpha:
+  approver: "@wojtek-t"
+beta:
```

Comment:
If you really want to go directly to beta here I would expect a paragraph in the KEP explaining the rationale and potential risks.

Reply:
sorry, no we don't target beta for 1.30 - let me correct